### ISSUE TITLE
Merge Facility Owner and COA Division Info for EX & RW Folder (NN)

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#2397]
### As a ROW permit analyst, I'd like the Facility Owner Info and COA Division info merged into one info. I would like to keep the facility Owner Info and delete the COA Division info once the information has been transferred over into the Facility Owner Info.
**- To be worked with #1910**

**StakeHolders**: Jaclyn L, Isaiah L, Paloma
**DTS Manager:** Nate

**Acceptance Criteria:**
- Merge Franchise Owners/Project Sponsors Info and COA Division Info into one (See List attached) Infos: 76765 & 75943
- Keep Franchise Owners/Project Sponsors Info. Delete COA Division info, Delete the Duplicates
- They need to be added to the EX for all SubTypes
- They need to be added to the  RW folder for SubType TURP
- On the EX AND RW TURP folder:
			- Remove COA Division Info
			- Change mandatory info name from **_Franchise / COA Department_** and **_Facility Owner/project Sponsor_** to **_Project 
                           Sponsor_**

**Updated Pick Values List:**
- Attached in comments

**Notes:**
- EX & RW folder: Infos are 76765 & 75943. 
- They just need one field, keep 76765 (Facility Owner/Project Sponsor), and update the pick values with list attached (More revisions to come).

**Comment by TracyLinder** [comment]
@Nadin-Nader I added you to this one as DTS Manager. Kaarthy will be scoping it during sprint 29.

**Comment by licarir** [comment]
This 75943 is on RW and EX folders currently.

**Comment by TracyLinder** [comment]
@kaarthym5 I'm updating the issue pipeline back to 'Needs Scoping'. Please use the 'Backlog' pipeline when the acceptance criteria is complete and the story is ready for development. Thank you.

**Comment by kaarthym5** [comment]
Hello Tracy, understood. We are currently working on refining this requirement and waiting for an update will move it to backlog once that’s complete.

Thanks
________________________________

**Comment by kaarthym5** [comment]
@nadin-Nader we would need this updated list to complete this merge request. I have also set up a meeting with the stakeholders to get the final requirement refinement and move this to the backlog.

**Comment by Nadin-Nader** [comment]
@kaarthym5 @TracyLinder List has been added and so has the acceptance criteria. Moving to Backlog.

**Status**
Ready to start work

**Comment by TracyLinder** [comment]
**Status:** Nate to get final OK on pick list values from stakeholders, then will be ready for dev.

**Comment by Nadin-Nader** [comment]
Hello Everyone,

Hope you are doing well. I was reaching out regarding 2397 Merge Info fields, EX & RW Folder to see if you were able to get the updated list for us. I have the list we went over in our meeting but if I remember correctly, there was more to be added or verified.

I have attached the most current list I have. It shows anything in RED is a modification or addition. Please review it and let me know if this is the final list and we can proceed with Development work.

Respectfully,

Nadin A. Nader 
IT Business Systems Analyst
Data and Technology Services 
Austin Transportation Department

**Comment by Nadin-Nader** [comment]
Email Response:

> From: Lozano, Jaclyn <Jaclyn.Lozano@austintexas.gov> 
> Sent: Thursday, September 17, 2020 3:54 PM
> To: Amayo-Ryan, Paloma <Paloma.Amayo-Ryan@austintexas.gov>; Nader, Nadin <Nadin.Nader@austintexas.gov>; Lewallen, Isaiah <Isaiah.Lewallen@austintexas.gov>; Perez, Kimberly <Kimberly.Perez@austintexas.gov>; Pepper, Gregory <Gregory.Pepper@austintexas.gov>
> Subject: RE: 2397 Merge Info fields, EX & RW Folder Follow up
> 
> Paloma, 
> 
> No I have not reached out to DSD. Yes that would be great if you can forward it to them. Thank you
> 
> Jaclyn Lozano | 1501 Toomey Rd. Austin,TX 78704 | Jaclyn.Lozano@AustinTexas.gov | 512 974 2412
> 
> From: Amayo-Ryan, Paloma 
> Sent: Wednesday, September 16, 2020 6:59 PM
> To: Nader, Nadin <Nadin.Nader@austintexas.gov>; Lozano, Jaclyn <Jaclyn.Lozano@austintexas.gov>; Lewallen, Isaiah <Isaiah.Lewallen@austintexas.gov>; Perez, Kimberly <Kimberly.Perez@austintexas.gov>; Pepper, Gregory <Gregory.Pepper@austintexas.gov>
> Subject: RE: 2397 Merge Info fields, EX & RW Folder Follow up
> 
> Good evening Nate,
> 
> I believe I completed all additions and edits that I had the last time we met.  As I recall we had discussed the need to confer with the General Permit staff regarding the acronyms.  Jaclyn, did you already reach out to them?  If not, I’d be happy to forward the list for their review.
> 
> Thx!
> Paloma Amayo-Ryan
> Program Manager
> (512) 974-2841

**Comment by Nadin-Nader** [comment]
@licarir @TracyLinder Had the meeting with Paloma, Kim, Neil and Sam. Neil informed us that the best people to talk to about this at Utility coordination and General Permitting, Zack Martin and/or Roderick Burns. He also informed us that they dont really care about the description but to check with them anyway just in case.
Paloma sent an email to Roderick Burns. If she doesnt get a response from Roderick, Paloma will make the changes needed and give us the green light to proceed with the development work.

**Waiting on Response from Roderick and/or Paloma**

**Comment by TracyLinder** [comment]
Per Paloma, she has not received a response from Roderick Burns. She's given the go-ahead to proceed without his input.

**Comment by TracyLinder** [comment]
Updated input list here:
[https://app.zenhub.com/files/140626918/24c667e5-662c-4513-9bd3-045041adf693/download](https://app.zenhub.com/files/140626918/24c667e5-662c-4513-9bd3-045041adf693/download)

**Comment by TracyLinder** [comment]
@Nadin-Nader can you please add a new value to the list you're maintaining? Paloma requested we add 'Project Connect' to the list. Thank you.

**Comment by TracyLinder** [comment]
@Nadin-Nader What will the name of the merged field be?

**Comment by TracyLinder** [comment]
@Nadin-Nader looking at the excel sheet of final values; it's hard to understand. Can we look at this?

**Comment by Nadin-Nader** [comment]
## **Next Step:**

Schedule time with Paloma to go over the Excel sheet

**Comment by TracyLinder** [comment]
@Nadin-Nader can you please ask stakeholders if they are using any reports that pull in either of these infos?

**Comment by Nadin-Nader** [comment]
Sent email to Paloma and Jaclyn:

From: Nader, Nadin 
Sent: Wednesday, April 7, 2021 3:58 PM
To: Lozano, Jaclyn <Jaclyn.Lozano@austintexas.gov>; Amayo-Ryan, Paloma <Paloma.Amayo-Ryan@austintexas.gov>
Cc: Lewallen, Isaiah <Isaiah.Lewallen@austintexas.gov>; Pepper, Gregory <Gregory.Pepper@austintexas.gov>; Perez, Kimberly <Kimberly.Perez@austintexas.gov>
Subject: RE: 2397 Merge Info fields, EX & RW Folder Follow up

Hi Paloma/Jaclyn,

Hope you are both doing well. I was reaching out to see if you have heard back from the General Permit staff regarding the acronyms? I was also wondering if you were aware of any reports that would be affected by the merging of the lists?

Please let me know as soon as you can so that we can have the developers start work on making the changes.
I would also like to schedule some time to go over the spelling and how you would like the name s to display one last time after we get the response from the General Permit department.

## **Status:**
Waiting on response

**Comment by Nadin-Nader** [comment]
Had the meeting with Paloma, Kim, Jaclyn Lozano, Kristin Carlton, Subhashini Shroff, Zackary Martin, John Maldonado, and Roderick Burns today (4/12/21) and went over the COA Division and Facility Owner List. We were able to identify the abbreviation and Descriptions needed as well as what needed to be added or removed from the list.
We were able to get most (95%) of the list completed. The last 5%, we will need to have another meeting with Paloma, Kim and one other person to determine if we need to add a few more.

New list can be located [here](https:\\coacd.org\dfs\C:\Users\NADERN\Desktop\FACILITY & COA OWNER stuff\COA and Facility Owner_4_12_21.xlsx)

## **Status :**
Waiting for meeting invite from Paloma

**Comment by Nadin-Nader** [comment]
**Meeting Scheduled for 4/14/21 @ 3pm** to go over the last bit of the list before we start working on it.

**Comment by Nadin-Nader** [comment]
@TracyLinder can you give me some input on if the Fee calculation would have a negative short term or long term effect (other than us trying to fix the calculation later on)?

@SubhaShroff Here are her responses from the question you had. 


> From: Nader, Nadin <Nadin.Nader@austintexas.gov> 
> Sent: Monday, April 12, 2021 3:17 PM
> To: Amayo-Ryan, Paloma <Paloma.Amayo-Ryan@austintexas.gov>; Perez, Kimberly <Kimberly.Perez@austintexas.gov>
> Subject: #2397 Merge COA Division and Facility Owners Info codes
> 
> 
> Hi Kim/Paloma,
> 
> Hi. I just got a few questions
> 
> 1.	Is the info code 75943-COA Division used for any fees insertion? Subha found this info code used in fee calculation/insertion code (COA_FOLDER_FEES Package).  **_This is news to me!_**
> 2.	How would this affect the fee calculation?  **_Given how inaccurate the data is in this list, and how often we skip past this field all together, I would be highly surprised to learn that it would affect fee calculation.  That said, AMANDA has surprised me a time or two in the past. 😃_** 
> 3.	I am not sure but Is 75943-COA Division used for any reports?  **_It is not used for any reports._**
> 
> Respectfully,
> 
> Nadin A. Nader (He/Him/His)
> IT Business Systems Analyst
> Data and Technology Services
>

**Comment by TracyLinder** [comment]
@Nadin-Nader I'm not 100% understanding what is being conveyed in the above comment, but I do have a tiny bit of insight. 

On TURPS, the fees only work if the value in info `Franchise / COA Department` is `'PVT'`. Any other value in that info, and the fees on the insert + run menu won't work. i.e. Staff has to manually insert the fee. The post-payment feature also looks at this info when the applicant submits via portal. If the info value is `PVT`, the system knows to insert + bill the application fee.

EX is similar, except the insert + run menu will still work. For EX, when the system sees PVT, it auto-inserts the application fees.

**Comment by SubhaShroff** [comment]
For reference I am adding the code, where I found 75943 used to calculate RW fee.:

The following is from COA_FOLDER_FEES package code:
![image](https://user-images.githubusercontent.com/51680715/114611716-f3741380-9c66-11eb-85c9-50cdab7f927d.png)
![image](https://user-images.githubusercontent.com/51680715/114611947-25857580-9c67-11eb-9146-487c58de9a27.png)

**Code in COA_FOLDER_FEES.f_folder_flat_fee FUNCTION:**

          WHEN pn_sub = ### gn_sub_turp THEN
                     --  Application fee
                     IF an_lookup1 = 300 THEN
                     ###    lv_insert_fee := coa_folder_fees.f_insert_fee_flag(pv_foldertype
                                                                          ,an_folderrsn);
                        IF lv_insert_fee = 'Y' THEN
                           lv_business_retention := coa_folder.f_get_info_string(an_folderrsn
                                                                                ,gn_business_retention);

                           IF pn_work = gn_work_pvt_short_term THEN
                              IF lv_business_retention = 'Yes' THEN
                                 ln_lookup1 := an_lookup1;
                              ELSE
                                 ln_lookup1 := an_lookup1 + 100;
                              END IF;
                           ELSIF pn_work = gn_work_pvt_long_term THEN
                              IF lv_business_retention = 'Yes' THEN
                                 ln_lookup1 := an_lookup1 + 200;
                              ELSE
                                 ln_lookup1 := an_lookup1 + 400;
                              END IF;
                           ELSE
                              ln_lookup1 := NULL;
                           END IF;
                           lv_error := NULL;
                        ELSIF lv_insert_fee = 'N' THEN
                           ln_lookup1 := NULL;
                        ELSE
                           lv_error := lv_insert_fee;
                        END IF;
                     ELSE
                        NULL;
                     END IF;

**Comment by SubhaShroff** [comment]
@TracyLinder  @Nadin-Nader In the above code 75943  is used to check if the selected COA Division is valid before inserting the fee. I think just replacing 75943 with 76765 will sort it.

**Comment by SubhaShroff** [comment]
@Nadin-Nader @TracyLinder  **### 76765** infocode should it be mandatory for RW TURP as in EX? 
And should it display in ABC Portal for RW TURP? I assume it should work the same as in EX.

**Comment by SubhaShroff** [comment]
Based on April 15th/16th chat on Teams with @Nadin-Nader(verified with stakeholders), the above questions are answered as Yes.

1.  #76765 infocode should it be mandatory for RW TURP as in EX? --** #YES**
And should it display in ABC Portal for RW TURP? I assume it should work the same as in EX. - **YES.**

**Comment by Nadin-Nader** [comment]
@SubhaShroff the answer to both would be Yes, per Kim.

**Comment by SubhaShroff** [comment]
**Status:** The changes are done on local branch, waiting for the remaining list items to be updated.

**Comment by Nadin-Nader** [comment]
The list has been updated but still needs about 8 more changes to it. Paloma said to go ahead and make the current changes and to schedule time with or send an email to Jaclyn or Isaiah to determine the rest. The most updated list is below:

[https://app.zenhub.com/files/140626918/258123c6-394f-478d-b99d-283f34367822/download](https://app.zenhub.com/files/140626918/258123c6-394f-478d-b99d-283f34367822/download)

**Comment by Nadin-Nader** [comment]
@SubhaShroff the list has been updated again but for the lat time. This time was to address the ones at the bottom of the list.

The new list to go by is below:

[https://app.zenhub.com/files/140626918/fa91d6b1-402a-40f6-83d3-059d3c625e38/download](https://app.zenhub.com/files/140626918/fa91d6b1-402a-40f6-83d3-059d3c625e38/download)

**Comment by SubhaShroff** [comment]
**Status:** I need to update the list on AMANDA as per the new list.

**Comment by TracyLinder** [comment]
**Notes from Demo**

- [ ] Rename the Info Name: Project Sponsor
- [ ] On TURP, remove info Franchise/COA Department
- [ ] Add: "PCO   Project Connect" to the info value list

@Nadin-Nader & @SubhaShroff FYI

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder 'Franchise/COA Department' info is only used by RW TURP. Since it is no more needed here also, should I completely wipe it off from AMANDA?

**Comment by SubhaShroff** [comment]
![image](https://user-images.githubusercontent.com/51680715/116929360-ecb43d00-ac23-11eb-8d0c-c24d8d153341.png)
![image](https://user-images.githubusercontent.com/51680715/116929372-f2118780-ac23-11eb-978b-57f82ebaddce.png)
![image](https://user-images.githubusercontent.com/51680715/116929428-06ee1b00-ac24-11eb-995a-8c7eba6accd7.png)
![image](https://user-images.githubusercontent.com/51680715/116929441-0b1a3880-ac24-11eb-8336-c971ccda89d8.png)


Hi @TracyLinder @Nadin-Nader  please see the above screenshots(aligned the list). 

And removed the info Franchise/COA Department from RW TURP, added PCO Project Connect

**Comment by SubhaShroff** [comment]
### Status:
New changes are done for updated AC, in local branch. I will post the demo screenshots soon.

**Comment by SubhaShroff** [comment]
![image](https://user-images.githubusercontent.com/51680715/117151285-8c85de00-ad7e-11eb-9ace-7bce0d5aaa7f.png)


### **RW TURP**
![image](https://user-images.githubusercontent.com/51680715/117151499-c525b780-ad7e-11eb-9046-1c5382c03b8f.png)

**Comment by SubhaShroff** [comment]
### Status:
Latest AC updates are implemented on local branch. 
Await the feedback from @TracyLinder @Nadin-Nader.

**Comment by SubhaShroff** [comment]
DEV Promotion request is submitted - **SCTASK0136583** (for commit-cb52855393b6)

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder 
I tested in DEV and above are the screenshots from DEV. 
• Project Sponsor pick list alignment,
• PCO Project Connect – added
• Name changed to ‘Project Sponsor’
• ‘Franchise/COA Department’ – removed from RW TURP
Removed 'COA Division'

**Comment by SubhaShroff** [comment]
### Status:
Tested today on DEV. Waiting for feedback. The screenshots are provided above.

**Comment by Nadin-Nader** [comment]
@SubhaShroff It looks good on DEV. You can go ahead and promote it to TEST.


**Status:**

## **Approved for TEST Deployment 🎉🙌**

**Comment by SubhaShroff** [comment]
@Nadin-Nader  TEST deployment request is submitted - **SCTASK0137648**(for commit-cb52855393b6)

**Comment by Nadin-Nader** [comment]
@Nadin-Nader UAT ON PROD MONDAY

**Comment by SubhaShroff** [comment]
@Nadin-Nader I tested on BO. Following are the screenshots from TEST:
### EX
![image](https://user-images.githubusercontent.com/51680715/118333824-0aaa5900-b4d2-11eb-9fec-7e6cfc3a05e4.png)

### RW TURP
![image](https://user-images.githubusercontent.com/51680715/118334119-8e644580-b4d2-11eb-8676-1b13a2ed68d7.png)
![image](https://user-images.githubusercontent.com/51680715/118334152-a3d96f80-b4d2-11eb-97c6-9d180818178e.png)

Portal changes, I will test once Swapna does clear cache.

**Comment by SubhaShroff** [comment]
@Nadin-Nader 

### 2021-038548 EX
![image](https://user-images.githubusercontent.com/51680715/118545876-cf4f9a80-b71c-11eb-8291-0af6b8500bbf.png)


![image](https://user-images.githubusercontent.com/51680715/118547099-83055a00-b71e-11eb-9c9c-7e37e0e1bb89.png)

RW TURP - Failed in Folder Info validation for 'COA Department'. I will have to take care of that. The same error occurred on Portal also.
![image](https://user-images.githubusercontent.com/51680715/118548690-302ca200-b720-11eb-81b7-96c76be5675e.png)

**Comment by SubhaShroff** [comment]
@Nadin-Nader Today I investigated about the above error. It is actually a validation error raised for RW TURP while fees insertion as part of auto add cycle after saving the infos on info tab. This validation checks for both infos 75942, 75943(AC is to remove both), before inserting RW TURP fees (50165, 50220).

1.  Should I replace this validation with the validation for 76765(Project Sponsor) check? 
2. Or Since 76765 is mandatory field, just take out that validation for  TURP fee insertion and add nothing?

**Comment by SubhaShroff** [comment]
### Status:
Tested on TEST. Failed due to validation check. The error is displayed, while TURP fees auto insertion. Expecting inputs on it for further proceeding.

**Comment by Nadin-Nader** [comment]
I spoke to Paloma via chat and she said she prefers option 1. @SubhaShroff Please go ahead with Option 1 per Paloma. Just FYI (in the future) we may have to revisit this.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  A commit( 03831069c704) is done for the validation replacement with 76765 info check for RW TURP fees insertion as per option 1. I created a RW TURP on local branch and it didn't throw any error, after the code change.  The permit was created successfully. Please audit the commit to move forward.

![image](https://user-images.githubusercontent.com/51680715/118877681-584d0a00-b8b4-11eb-88f3-ecd77e5c28dc.png)

**Comment by SubhaShroff** [comment]
@Nadin-Nader  **SCTASK0138949** - DEV deployment request is submitted for the commit-03831069c704, to replace the validation with 76765 validation for TURP fees insertion.

**Comment by Nadin-Nader** [comment]
**Status:**

Waiting on CTM to deploy to DEV

**Comment by Nadin-Nader** [comment]
@SubhaShroff this has been deployed to DEV

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I tested the changes on **DEV** for 
- **RW TURP (11901148)** - created and issued.
- EX (11901149) - Created Permit.

**Comment by SubhaShroff** [comment]
### Status:
Ready for DEV testing.

**Comment by Nadin-Nader** [comment]
@SubhaShroff Looks good on Dev. Lets move forward to TEST and have the Stakeholders UAT in TEST.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I submitted a request **SCTASK0140837** (commit-03831069c704) to do TEST deployment.

**Comment by SubhaShroff** [comment]
TEST deployment is done, I need to test it yet.

**Comment by Nadin-Nader** [comment]
Subha is going to test her changes to make sure its ready for UAT

**Comment by Nadin-Nader** [comment]
We were able to test this in our meeting today with ROW (Izzy) and it did not pass as the PVT option in the list is not generating the TURP Right of Way Usage fee (old info # 75942) when the Event Review is accepted.
Subha is going to look over the code to see whats going on.

**Status:** 
Not approved for PROD

**Comment by SubhaShroff** [comment]
@Nadin-Nader  necessary changes are done in local and demo video is sent in email (6/7/2021).

**Comment by SubhaShroff** [comment]
### Status:
Waiting for the feedback to do the new commit.

**Comment by Nadin-Nader** [comment]
@SubhaShroff i have reviewed this and saw that the changes have been made and the fee is calculating the dimensions. Go ahead and do the commit for this to go to DEV

## **Status:**
Ready for Dev Deployment.🙌🎉

**Comment by SubhaShroff** [comment]
@Nadin-Nader I did the commit for the code change - **1406a8b67ad5**. Please audit this commit when you are back.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  Thanks for approving the commit. I submitted the DEV deployment request for this commit  - **SCTASK0145063**

**Comment by SubhaShroff** [comment]
@Nadin-Nader  Dev deployment is done - SCTASK0145063. Following RW TURPs are created and I found couple of fees got inserted after TUPR Review approved.
11901309
11901310
This ticket is ready for your testing on DEV.

**Comment by SubhaShroff** [comment]
### Status:
I tested it on DEV and its ready for DEV testing.

**Comment by SubhaShroff** [comment]
@Nadin-Nader @TracyLinder While looking into TURP Permit Extension process, I came across COA FRANCHISE(**75942**). That with EXTENSION GRANTED attempt, if 75942 is PVT%,  permit gets ACTIVated. Then I checked for this infocode use and I found it used in Payment procedure as well.
##--YET TO ADD MORE DETAILS of Payment procedure

**Comment by Nadin-Nader** [comment]
@SubhaShroff was there more you were going to add?

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I completed the code changes to replace 75942 with 76765 in Payment procedure and Pemit Extension processes. I have given you the commits-  e29b28b444d7 and ab1278cf3d0b  for audit.

**Comment by SubhaShroff** [comment]
DEV deployment of commits (e29b28b444d7 and ab1278cf3d0b)   is submitted - **SCTASK0146532**.

**Comment by Nadin-Nader** [comment]
@SubhaShroff George has completed the DEV deployment on this.

**Comment by SubhaShroff** [comment]
Thank you @Nadin-Nader. I tested this for TURP - 11901428 until activation. I didn't find any errors. And it looks good to me. Its up for your testing.

**Comment by SubhaShroff** [comment]
### Status:
Up for review in DEV, testing for the entire TURP cycles would help.

**Comment by SubhaShroff** [comment]
### Status:
Waiting on DEV review to move to TEST.

**Comment by Nadin-Nader** [comment]
@SubhaShroff Looks good. Lets get this over to TEST, UAT and then have the stakeholders UAT or approve the move to PROD

**Comment by SubhaShroff** [comment]
@Nadin-Nader TEST deployment is submitted for commits (1406a8b67ad5,  e29b28b444d7, ab1278cf3d0b) - **SCTASK0149317**

**Comment by SubhaShroff** [comment]
### Status:
TEST deployment is done and I need to test it.

**Comment by Nadin-Nader** [comment]
**Status:**
UAT on TEST

**Comment by Nadin-Nader** [comment]
Subha informed me yesterday that she is still working on this and that the fees are still not being added in DEV.
Subha will be back on the 15th of July.

**Comment by Nadin-Nader** [comment]
**STATUS:**
Subha would like to test/discuss with Kim at the meeting today

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I have sent you the demo screens today in email. Please review.

**Comment by SubhaShroff** [comment]
### Status:
Waiting for feedback to move to TEST.

**Comment by Nadin-Nader** [comment]
@SubhaShroff i reviewed the demo you sent and it looks good. lets move this along

**Comment by SubhaShroff** [comment]
@Nadin-Nader  The code is already in place in TEST and I retested it for EX & RW TURP. Its ready for your UAT.
12658729  EX
12602795 RW TURP

**Comment by SubhaShroff** [comment]
### Status:
Up for UAT.

**Comment by Nadin-Nader** [comment]
@SubhaShroff demo screenshots look good but i set them to Paloma to approve the move to PROD

**Status:**
Waiting on Paloma's response

**Comment by Nadin-Nader** [comment]
@Nadin-Nader Go over in Fridays meeting

**Comment by TracyLinder** [comment]
**Conversation w/ Paloma**

She is asking for a demo of this. @Nadin-Nader can you please schedule the demo for monday or tuesday?

**Comment by Nadin-Nader** [comment]
@TracyLinder @SubhaShroff Scheduled meeting for the 4th of August 2021 at 11am

**Comment by Nadin-Nader** [comment]
@SubhaShroff Paloma has approved the move to PROD for this request.

**Comment by Nadin-Nader** [comment]
## _**Approved for deployment to PROD🎉🙌‼**

**Comment by SubhaShroff** [comment]
@Nadin-Nader PROD Deployment request is raised - **SCTASK0157638**.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  PROD deployment is done. 🎉✨

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0159438 is submitted to restore the VALIDINFO values for the old infos. Those are still needed for old RW-TURPs that are created before the deployment of the new info in PROD.

**Comment by SubhaShroff** [comment]
### Status:
In Progress.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  **SCTASK0160583** - Submitted DEV promotion request for e86b804508dc.

**Comment by SubhaShroff** [comment]
### Status:
After call with Gita, based on her suggestion have done a  change in configuration to display the old infos(75942. 75943 if are with permit). I will commit it today and submit DEV promotion for it. The change can be tested only in DEV.

**Comment by TracyLinder** [comment]
Note: During UAT we need to test incomplete existing folders. We can do this in test.

**Comment by Nadin-Nader** [comment]
**Status:**
UAT Session scheduled for Tuesday 8/31/21 with Paloma, Kim, Izzy and @SubhaShroff

**Comment by Nadin-Nader** [comment]
UAT Scheduled for 9/2/21 @ 930am

**Comment by Nadin-Nader** [comment]
**Status:**
Subha will address the issue with old permits and having to add the new project sponsor field. Kim said that is not a deal breaker but Subha is going to see if she can fix it.

**Comment by SubhaShroff** [comment]
### Status:
TURP Application Fee is not inserting for old infos (75942, 75943). I am going to work on this today.

**Comment by SubhaShroff** [comment]
@Nadin-Nader SCTASK0165895 - is submitted for DEV deployment of a correction that was not allowing to insert TURP Application Fee and ATD surcharge fee.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I tested it for RW-TURP pvt, by doing 'Insert + run' for TURP Application Fee on DEV. Now it adds the fee.
Please let me know if we can push it further.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0167083 is submitted with the TEST deployment plan - for a correction that was not allowing to insert TURP Application Fee and ATD surcharge fee.

**Comment by Nadin-Nader** [comment]
This was fixed and is now on TEST awaiting @SubhaShroff UAT before we UAT with Stakeholders for approval to PROD.

**Status:**
Subha is testing on TEST

**Comment by SubhaShroff** [comment]
### Status:
Tested on TEST with Incomplete application with old infos and submitted. TURP Application Fee & ATD Surcharge Fee added.
RW - 12562049.

**Comment by SubhaShroff** [comment]
### Status:
Ready for UAT on TEST.

**Comment by Nadin-Nader** [comment]
Status:
Tested on TEST with Incomplete application with old infos and submitted. TURP Application Fee & ATD Surcharge Fee added.
RW - 12562049.
Tested for RW - 12562049 (new permit application.) Fees inserted with submission.

**Status:**
Ready for UAT on TEST.

**Comment by Nadin-Nader** [comment]
Message from the chat with Paloma on 9/27/21:

> [Yesterday 4:43 PM] Nader, Nadin
>     
> it is, sorry for the confusion. Have yall had a chance to UAT 2397 (Merge Facility Owner and COA Division Info for EX & RW Folder)?
> ​[Yesterday 4:52 PM] Amayo-Ryan, Paloma
>     No UAT but we did see the demo on Friday (I believe it was Friday?) and it looked good to me!
> ​[Yesterday 4:56 PM] Nader, Nadin
>     ok so do you mind if we deploy it to PROD?
> ​[Yesterday 5:12 PM] Amayo-Ryan, Paloma
>     Not all; I say it's good to go!

**Comment by Nadin-Nader** [comment]
##  APPROVED for **_PROD_** deployment👍🙌🎉‼

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder  SCTASK0169891 - submitted PROD deployment request.

**Comment by Nadin-Nader** [comment]
**Status:**
Waiting on DevOps CM Team to deploy to PROD

**Comment by Nadin-Nader** [comment]
Per Paloma on Oct 1, 2021 this seems good to go on PROD. CLOSING CARD
